node/test/@opent2t/test-b/TranslatorBC.js

```diff
@@ -1,78 +0,0 @@
-"use strict";
-// This code uses ES2015 syntax that requires at least Node.js v4.
-// For Node.js ES2015 support details, reference http://node.green/
-
-const EventEmitter = require("events");
-
-/**
- * This translator class implements two interfaces: "org.opent2t.test.B"
- * and "org.opent2t.test.C". Since those interfaces do not have any conflicting
- * member names, they are both implemented directly.
- */
-class TestTranslatorBC extends EventEmitter {
-
-    constructor(deviceProps) {
```

Comment:
Note this code is red, meaning it was deleted. :)

But I didn't switch to initDevice() in the new test translator code. That's orthogonal to the rest of these changes anyway, so we can change that later, or not.